tests/components/sql/test_config_flow.py

```diff
@@ -266,7 +266,6 @@ async def test_options_flow_name_previously_removed(
         "db_url": "sqlite://",
         "query": "SELECT 5 as size",
         "column": "size",
-        "value_template": None,
```

Comment:
why is it being removed ?

Reply:
As it's optional and therefore if the user leaves the field empty it's not stored anymore, it would remove the option completely from the options dict.
This is the original fix in this PR as in previous setup it was impossible to have a value template and later wanting to remove it, same for UoM.